client/scss/components/_page_description.scss

```diff
@@ -0,0 +1,66 @@
+//* @define page-description;
+
+.page-description {
+  &.page-description--wb {
```

Comment:
yeah, you're right – in searching for what set the two variants apart the most, it was the font-family that was the winner, but this isn't ideal, granted.